mw-webapp/src/dataAccessLogic/WayPreviewDAL.ts

```diff
@@ -14,66 +14,47 @@ export class WayPreviewDAL {
    */
   public static async getWaysPreview(): Promise<WayPreview[]> {
     const waysDTO = await WayService.getWaysDTO();
-    const usersPreview = await UserPreviewDAL.getUsersPreview();
-    const goalsPreview = await GoalPreviewDAL.getGoalsPreview();
-
-    const ownersPreview = waysDTO.map((wayDTO) => {
-      const ownerPreview = usersPreview
-      //TODO: task #114 Use hashmap instead of .find
-        .find((elem) => elem.uuid === wayDTO.ownerUuid);
-      if (!ownerPreview) {
-        throw new Error(`${ownerPreview} was not found`);
-      }
-
-      return ownerPreview;
-    });
+    const waysUuids = waysDTO.map((item) => {
+      const wayUuid = item.uuid;
```

Comment:
can you tell me why to declare a variable here, and not immediately return the result

Reply:
Because I always try to return var, not just code. Do we have rules about it? If you want we can discuss about it in the group and add some rules about it))

Reply:
But yes I agree in this case we can return uuid without creating vars)

Reply:
But yes I agree in this case we can return uuid without creating vars)